imagecodecs error while building tomojs-pytools

While attempting to build tomojs-pytools on an M1 Mac running Monterey Version 12.6.2, I get the following error:
```
imagecodecs/_aec.c:756:10: fatal error: 'libaec.h' file not found
      #include "libaec.h"
               ^~~~~~~~~~
      1 warning and 1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> imagecodecs
```

Same error with both conda Python 3.9.15 and Homebrew Python 3.10.10

Hardware Overview:
```
 Model Name:	MacBook Pro
  Model Identifier:	MacBookPro18,1
  Chip:	Apple M1 Pro
  Total Number of Cores:	10 (8 performance and 2 efficiency)
  Memory:	16 GB
  System Firmware Version:	8419.60.44
  OS Loader Version:	7459.141.1
  Serial Number (system):	TNV664X05Y
  Hardware UUID:	B543DCDB-F3E3-51D8-88B2-6860F79AAF3A
  Provisioning UDID:	00006000-000C04C11A02401E
  Activation Lock Status:	Disabled
```

See attached file for complete run.
[imagecodes_error.txt](https://github.com/niaid/tomojs-pytools/files/10724825/imagecodes_error.txt)

--- a/README.rst
+++ b/README.rst
@@ -24,4 +24,6 @@ the installation process installs the scripts as executables.
 
 Alternatively, the package can be installed from the git repository:
 
-`python3 -m pip https://github.com/niaid/tomojs-pytools.git@v0.6`
+`python3 -m pip install git+https://github.com/niaid/tomojs-pytools.git@v1.3.2`
+
+**Note for Mac M1 development**: The imagecodec package does not provide a Mac OSX ARM binary at this time. The current workaround is to use an x86_64 python version on the M1 Mac in order to use the exiting binaries on PyPi.

